app/views/menu_days/_menu.html.haml

```diff
@@ -1,17 +1,43 @@
-.soups
-  %h3= 'Soups'
-  %ul
-    - dishes.soup.each do |soup|
-      %li= soup.name
+= simple_form_for Order.new, url: new_order_path, method: :get do |f|
```

Comment:
method: :post I guess.
Read about http methods
Get - only to get data, not change
Post - for creating as a rule.
Put or Patch for update
Delete for destroy

Reply:
@VolodymyrZyblikevich